apps/demo/translations/src/en/validation.json

```diff
@@ -46,5 +46,14 @@
   },
   "appealStages": {
     "required": "Select an appeal stage from the list"
+  },
+  "saveForm": {
+    "required": "Select an option below"
```

Comment:
Select if you want to save your form or not.

Reply:
changed line 51 to 'Select if you want to save your form or not.'